modal/aio.py

```diff
@@ -44,3 +46,9 @@
     "aio_web_endpoint",
     "aio_wsgi_app",
 ]
+
+deprecation_warning(
+    datetime.date(2023, 5, 12),
+    "The modal.aio module and Aio* classes will soon be deprecated. Use `await some_function.aio(...)` instead.",
```

Comment:
Might want to have an example for classes too?

Reply:
might not be necessary btw – i was confused and thought we went for the idea where classes have a special `aio` property